### ISSUE TITLE
RDS DBInstance Status never gets updated, stays in creating

### ISSUE THREAD

**Issue by JamesClonk** [aws-controllers-k8s/community#923]
**Describe the bug**
When using the rds-controller and creating a `DBInstance` resource, the status of that resource never gets updated beyond the initial creation. Even hours later after the RDS DB instance on AWS is long since `Available`, the status on the resource still shows `creating` and doesn't list any of the additionally expected properties like `Endpoint` for example. The rds-controller does not seem to update or reconcile DBInstance status after the initial createDBInstance call towards AWS RDS.

**Steps to reproduce**
- deploy rds-controller
- create a sample `DBInstance`:
```yaml
---
apiVersion: v1
kind: Secret
metadata:
  name: example-db-secret
  namespace: example
type: Opaque
stringData:
  password: super-safe-example-secret
---
apiVersion: rds.services.k8s.aws/v1alpha1
kind: DBInstance
metadata:
  name: example-db
  namespace: example
spec:
  allocatedStorage: 10
  dbInstanceClass: db.t3.micro
  dbInstanceIdentifier: example-db
  engine: postgres
  engineVersion: "13.2"
  masterUsername: root
  masterUserPassword:
    namespace: example
    name: example-db-secret
    key: password
```
- observe that the status never leaves `creating`, and Db Instance Port never changes to != 0:
```yaml
$ kubectl -n example describe dbinstance example-db
...
Status:
  ...
  Db Instance Port:           0
  Db Instance Status:         creating
...
```

**Expected outcome**
After the RDS DB instance reaches status `Available`on AWS it should reflect that status on the K8s `DBInstance` resource. The `DBInstance` resource should now contain a field `Endpoint` with the respective value, also `DBInstancePort` should not be empty / `0` anymore.

**Environment**
* EKS, Kubernetes version 1.20
* rds-controller, v0.0.5 deployed via Helm chart
* AWS RDS - Postgres

**Comment by JamesClonk** [comment]
I can see in the debug logs that there is a patchResourceStatus step being mentioned, but the last time this occured was *before* the status on AWS RDS was `Available`, but looking at the timestamps this seems to happen pretty much just right at the start of creating the DB Instance. After that there's nothing anymore in the logs, certainly not once the instance reaches `Available`.
```
2021-08-31T07:24:39.982Z	DEBUG	ackrt	> r.Sync	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:39.982Z	DEBUG	ackrt	>> rm.ReadOne	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:39.982Z	DEBUG	ackrt	>>> rm.sdkFind	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.026Z	DEBUG	ackrt	<<< rm.sdkFind	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.026Z	DEBUG	ackrt	<< rm.ReadOne	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "error": "resource not found"}
2021-08-31T07:24:40.027Z	DEBUG	ackrt	>> r.createResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.027Z	DEBUG	ackrt	>>> r.setResourceManaged	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.027Z	DEBUG	ackrt	>>>> r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.027Z	DEBUG	ackrt	>>>>> kc.Patch (metadata + spec)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.041Z	DEBUG	ackrt	<<<<< kc.Patch (metadata + spec)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.041Z	DEBUG	ackrt	patched resource metadata and spec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "latest": {}}
2021-08-31T07:24:40.041Z	DEBUG	ackrt	<<<< r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.041Z	DEBUG	ackrt	marked resource as managed	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.041Z	DEBUG	ackrt	<<< r.setResourceManaged	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.041Z	DEBUG	ackrt	>>> rm.Create	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:40.041Z	DEBUG	ackrt	>>>> rm.sdkCreate	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.176Z	DEBUG	ackrt	<<<< rm.sdkCreate	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.176Z	DEBUG	ackrt	<<< rm.Create	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.176Z	DEBUG	ackrt	>>> rm.ReadOne	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.176Z	DEBUG	ackrt	>>>> rm.sdkFind	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.311Z	DEBUG	ackrt	<<<< rm.sdkFind	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.311Z	DEBUG	ackrt	<<< rm.ReadOne	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.311Z	DEBUG	ackrt	>>> r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.311Z	DEBUG	ackrt	>>>> kc.Patch (metadata + spec)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	<<<< kc.Patch (metadata + spec)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	patched resource metadata and spec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "latest": {}}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	<<< r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	INFO	ackrt	created new resource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	<< r.createResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	>> r.lateInitializeResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	>>> rm.LateInitialize	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	no late initialization required.	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	<<< rm.LateInitialize	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	>>> r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	no difference found between metadata and spec for desired and latest object.	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	<<< r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	<< r.lateInitializeResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	< r.Sync	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	> r.patchResourceStatus	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.331Z	DEBUG	ackrt	>> kc.Patch (status)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.347Z	DEBUG	ackrt	<< kc.Patch (status)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.347Z	DEBUG	ackrt	patched resource status	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "latest": {}}
2021-08-31T07:24:41.347Z	DEBUG	ackrt	< r.patchResourceStatus	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 1}
2021-08-31T07:24:41.347Z	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "dbinstance", "request": "example/example-db"}
2021-08-31T07:24:41.348Z	DEBUG	ackrt	> r.Sync	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.348Z	DEBUG	ackrt	>> rm.ReadOne	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.348Z	DEBUG	ackrt	>>> rm.sdkFind	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	<<< rm.sdkFind	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	<< rm.ReadOne	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	>> r.updateResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	<< r.updateResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	>> r.lateInitializeResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	>>> rm.LateInitialize	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	no late initialization required.	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	<<< rm.LateInitialize	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	>>> r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	no difference found between metadata and spec for desired and latest object.	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	<<< r.patchResourceMetadataAndSpec	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	<< r.lateInitializeResource	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	< r.Sync	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	> r.patchResourceStatus	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.474Z	DEBUG	ackrt	>> kc.Patch (status)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.489Z	DEBUG	ackrt	<< kc.Patch (status)	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.489Z	DEBUG	ackrt	patched resource status	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "latest": {}}
2021-08-31T07:24:41.489Z	DEBUG	ackrt	< r.patchResourceStatus	{"kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2, "account": "1111111111", "role": "", "region": "eu-central-1", "is_adopted": false, "kind": "DBInstance", "namespace": "example", "name": "example-db", "generation": 2}
2021-08-31T07:24:41.489Z	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "dbinstance", "request": "example/example-db"}
```

Is there some way maybe to manually force the controller to do another / new reconcile loop?

**Comment by jaypipes** [comment]
@JamesClonk ty for the bug report. I'm looking into this right now.

**Comment by RedbackThomson** [comment]
Released as part of `v0.5.0`

**Comment by jaypipes** [comment]
aws-controllers-k8s/rds-controller#29 was never merged which is the fix for this issue.
